.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!--
+Before filling out an issue:
+1. A bug means that there is something broken or outside expectations in react-native-elements. If you only need help writing your own components, check out the [Slack channel](https://react-native-elements-slack.herokuapp.com/) FIRST.
+2. If any section is left blank, there is a high possibility that the ticket will be closed. If you want your issue to get solved, you need to fill out ALL of the below fields. Otherwise, the ticket will likely be closed as not providing enough info.
+-->
+
+**Explain what you did (Required)**
+
+<!-- What you were trying to accomplish. -->
+
+**Expected behavior (Required)**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Describe the bug (Required)**
+
+<!--  A clear and concise description of what the bug is. -->
+
+**To Reproduce (Required)**
+
+> We highly recommend that you re-create the bug on [Snack](https://snack.expo.io). If not, list the steps that a reviewer can take to reproduce the behaviour:
+
+<!--
+Example:
+
+```bash
```

Comment:
Why code block

Reply:
That's a typo... May i correct it

Reply:
@pranshuchittora , actually the steps.... need to be hidden, so only commenting the steps didn't worked.. I have wrapped the steps in a code block.

Reply:
@pranshuchittora , May I change or leave it as it is ?

Reply:
Remove the code block and it should be outside the comment as
1.
2.
3.

Like this without any content
```
1.
2.
3.
```

Reply:
ok will do it asap